android/src/main/java/com/mvcpscrollviewmanager/MvcpScrollViewManagerModule.java

```diff
@@ -50,65 +88,70 @@ public void initialize() {
   @ReactMethod
   public void enableMaintainVisibleContentPosition(final int viewTag, final int autoscrollToTopThreshold, final int minIndexForVisible, final Promise promise) {
     final UIManagerModule uiManagerModule = this.reactContext.getNativeModule(UIManagerModule.class);
-    this.reactContext.runOnUiQueueThread(new Runnable() {
```

Comment:
Is not needed to run the UI updates in a separate Thread since those always run in the UI thread. I can confirmed by calling `Looper.getMainLooper().isCurrentThread()` when updating the view.